### PR TITLE
Bug. Question type File. When multiple types are selected, only one is used.

### PR DESCRIPTION
**Pull request by AIlkiv**
```
foreach ($question['extraSettings']['allowedFileTypes'] as $type) {
	$question['accept'] = array_keys(array_filter($aliases, function (string $alias) use ($type) {
		return $alias === $type;
	}));
}
```
At each iteration, $question['accept'] is overwritten

### CODE REVIEW

File: lib/Service/FormsService.php [hunk 1]
```diff
@@ -121,14 +121,12 @@ public function getQuestions(int $formId): array {
 				$question['options'] = $this->getOptions($question['id']);
 				$question['accept'] = [];
 				if ($question['type'] === Constants::ANSWER_TYPE_FILE) {
-					if ($question['extraSettings']['allowedFileTypes'] ?? null) {
-						$aliases = $this->mimeTypeDetector->getAllAliases();
```
Comment (Chartman123) [inline lib/Service/FormsService.php:124]: Please remove the empty line

**Review by Chartman123 — COMMENTED** [review]
Some comments after reviewing the code, will test it later :)

File: tests/Unit/Service/FormsServiceTest.php [hunk 2]
```diff
@@ -61,6 +61,7 @@ function microtime(bool|float $asFloat = false) {
 use OCA\Forms\Service\CirclesService;
 use OCA\Forms\Service\ConfigService;
 use OCA\Forms\Service\FormsService;
+use OCP\AppFramework\Db\DoesNotExistException;
```
[the remark this answers is not in the document]
Comment (Chartman123) [reply tests/Unit/Service/FormsServiceTest.php:64]: oh, my bad... i just now noticed OCP vs OCA 😀

**Review by susnux — APPROVED** [review]

**Review by Chartman123 — APPROVED** [review]

**Comment by Chartman123** [comment]
@AIlkiv would you like to join our Forms team on cloud.nextcloud.com? If yes, I could help you get an account there as well...

**Comment by AIlkiv** [comment]
> @AIlkiv would you like to join our Forms team on cloud.nextcloud.com? If yes, I could help you get an account there as well...

Yes. Please help me with that

**Pull request merged**